applications/nrf_desktop/src/modules/hid_forward.c

```diff
@@ -879,7 +887,19 @@ static void disconnect_peripheral(struct hids_peripheral *per)
 				 &per->enqueued_reports);
 	__ASSERT_NO_MSG(!is_any_report_enqueued(&per->enqueued_reports));
 
+	/* Drop all the enqueued HID output reports. */
+	while (!sys_slist_is_empty(&per->enqueued_out_reports)) {
+		struct enqueued_out_report *item =
+			CONTAINER_OF(sys_slist_get(&per->enqueued_out_reports),
+				     __typeof__(*item),
```

Comment:
`__typeof__` seems to be very uncommon is such use case.
Maybe just,:
```suggestion
				     struct enqueued_out_report
```

Reply:
We already use it in other place. Changed here.